kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/sidePanels/QuizResourceSelection/subPages/QuestionsSettings.vue

```diff
@@ -0,0 +1,198 @@
+  import { PageNames } from '../../../../../../constants';
+  import { injectQuizCreation } from '../../../../../../composables/useQuizCreation';
+
+  /**
+   * @typedef {import('../../../../../../composables/useFetch').FetchObject} FetchObject
+   */
+
+  export default {
+    name: 'SelectFromBookmarks',
+    components: {},
+    setup(props) {
+      const prevRoute = ref(null);
+      const instance = getCurrentInstance();
+
+      const {
+        questionsSettingsLabel$,
+        numberOfQuestionsLabel$,
+        maxNumberOfQuestionsInfo$,
+        maxNumberOfQuestions$,
+        chooseQuestionsManuallyLabel$,
+      } = enhancedQuizManagementStrings;
+      const { activeSection, activeSectionIndex } = injectQuizCreation();
+
+      const invalidSettings = computed(
+        () =>
+          props.settings.questionCount > props.settings.maxQuestions ||
+          props.settings.questionCount < 1,
+      );
+
+      const questionCount = computed({
+        get: () => props.settings.questionCount,
+        set: value => {
+          instance.proxy.$emit('update:settings', {
+            ...props.settings,
+            questionCount: value,
+          });
+        },
+      });
+
+      props.setTitle(
+        questionsSettingsLabel$({
+          sectionTitle: displaySectionTitle(activeSection.value, activeSectionIndex.value),
+        }),
+      );
+      props.setGoBack(null);
+
+      const continueHandler = () => {
+        if (!props.isLanding && prevRoute.value) {
+          instance.proxy.$router.push(prevRoute.value);
+          return;
+        }
+        instance.proxy.$router.push({
+          name: PageNames.QUIZ_SELECT_RESOURCES_INDEX,
+        });
+      };
+      onMounted(() => {
+        props.setContinueAction({
+          handler: continueHandler,
+        });
+      });
+      watch(
+        () => props.settings,
+        () => {
+          props.setContinueAction({
+            handler: continueHandler,
+            disabled: invalidSettings.value,
+          });
+        },
+      );
+      onUnmounted(() => {
+        props.setContinueAction(null);
+      });
+
+      return {
+        // eslint-disable-next-line vue/no-unused-properties
+        prevRoute,
+        questionCount,
+        maxQuestions: computed(() => props.settings.maxQuestions),
+        maxNumberOfQuestions$,
+        numberOfQuestionsLabel$,
+        maxNumberOfQuestionsInfo$,
+        chooseQuestionsManuallyLabel$,
+      };
+    },
+    props: {
+      setTitle: {
+        type: Function,
+        default: () => {},
+      },
+      setGoBack: {
+        type: Function,
+        default: () => {},
+      },
+      setContinueAction: {
+        type: Function,
+        default: () => {},
+      },
+      settings: {
+        type: Object,
+        required: true,
+      },
+      isLanding: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    beforeRouteEnter(to, from, next) {
+      next(vm => {
```

Comment:
I'm getting this error, the initial stack trace points to here

![image](https://github.com/user-attachments/assets/fccd8852-0240-468b-9ae3-2e3fead566f2)

Reply:
This prevents me from adding any contents  as everything beneath the "[] Choose questions manually" string is blank

Reply:
Hmm thats weird, I am not getting it, Im not using a KRadioButton anywhere in that page :thinking:, and thats a warning of a KRadioButton validator, it shouldnt prevent us from seeing the rendered component :thinking:.

Are you seeing any other errors in the console? Could you send a screencast/screenshot of what are you seeing in the side panel?